requirements/base.in

```diff
@@ -18,7 +18,7 @@ edx_rest_api_client
 mysqlclient
 pytz
 python-dateutil==2.8.0    # botocore has a <2.8.1 constraint. This is not a direct dependency and can be deleted (vs unpinned) when no longer needed
-wagtail==2.3     # wagtail>2.3 dropped support for django111
+wagtail==2.6.2     # wagtail>2.3 dropped support for django111
```

Comment:
This upgrade drops support for Django 1.11; I don't want to merge this until we've already checked the deprecation warnings from the additional Travis workers added by https://openedx.atlassian.net/browse/BOM-1386 .

Reply:
[BOM-1384](https://openedx.atlassian.net/browse/BOM-1384) is dependent on this specific version. The edx-lint upgrade requires a fix that was released in this 2.6.2 release of wagtail. 
The tests are also passing on django111 with this change so shouldn't it be okay to merge this change?

Reply:
The comment here should be updated to explain why we're now pinned to 2.6.2.

Reply:
Tests are not passing with django111. Previously, the Django version was updated to 2.2.11 for a test and I forgot to revert it back to 1.11. 
I'll be adding a previous version in the requirements file for Django111 and will use the latest GitHub commit for Django2+ versions. I hope this can resolve our issues.